src/der.rs

```diff
@@ -77,19 +77,26 @@ pub(crate) fn expect_tag_and_get_value_limited<'a>(
     Ok(inner)
 }
 
+pub(crate) fn nested_limited<'a, R, E: Copy>(
+    input: &mut untrusted::Reader<'a>,
+    tag: Tag,
+    error: E,
+    decoder: impl FnOnce(&mut untrusted::Reader<'a>) -> Result<R, E>,
+    size_limit: usize,
+) -> Result<R, E> {
+    let inner = expect_tag_and_get_value_limited(input, tag, size_limit).map_err(|_| error)?;
```

Comment:
Style nit: not much point in binding inner here, can just chain.

Reply:
Good catch, fixed.